projects/plugins/jetpack/extensions/blocks/subscriptions/api.js

```diff
@@ -12,3 +12,16 @@ export function getSubscriberCount( successCallback, failureCallback ) {
 		}
 	} );
 }
+
+export function getSubscriberCounts( successCallback, failureCallback ) {
+	return apiFetch( {
+		path: '/wpcom/v2/subscribers/counts',
+	} ).then( count => {
```

Comment:
Thank you @mashikag for this suggestion. I added it to both API calls, and threw in a check for undefined, otherwise 0 results would also trip the `failureCallback` in `getSubscriberCount`:

```js
export function getSubscriberCount( successCallback, failureCallback ) {
	return apiFetch( {
		path: '/wpcom/v2/subscribers/count?include_publicize_subscribers=false',
	} ).then( ( { count } = {} ) => {
		// Handle error condition
		if ( typeof count !== 'undefined' ) {
			successCallback( count );
		} else {
			failureCallback();
		}
	} );
}

export function getSubscriberCounts( successCallback, failureCallback ) {
	return apiFetch( {
		path: '/wpcom/v2/subscribers/counts',
	} ).then( ( { counts } = {} ) => {
		// Handle error condition
		if ( counts ) {
			successCallback( counts );
		} else {
			failureCallback();
		}
	} );
}

```

Reply:
> threw in a check for undefined, otherwise 0 results would also trip the failureCallback in getSubscriberCount

What about: 
```
if ( Number.isFinite( count ) ) {
  // success
} else {
  // failure
}
```

Reply:
BTW you could also:
`if ( count !== undefined )`
without the need to use `typeof` :)

Reply:
You're 100% right :)